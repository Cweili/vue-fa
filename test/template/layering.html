<FaLayers
  size="4x"
  pull="left"
>
  <Fa :icon="faFlag" />
  <FaLayersText
    :scale="0.25"
    :rotate="-30"
    flip="both"
    color="white"
    style="font-weight: 900"
    size="lg"
  >
    NEW
  </FaLayersText>
</FaLayers>
