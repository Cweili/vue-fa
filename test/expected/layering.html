<span class="vue-fa-layers" style="float:left;width:1.25em;height:1em;font-size:4em;text-align:center;vertical-align:-.125em;transform-origin:center;overflow:visible;"><!--[--><svg viewBox="0 0 448 512" class="vue-fa" style="height:1em;vertical-align:-.125em;transform-origin:center;overflow:visible;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg"><g transform="translate(224 256)" transform-origin="112 0"><g transform="translate(0,0) scale(1,1) "><path d="M64 32C64 14.3 49.7 0 32 0S0 14.3 0 32V64 368 480c0 17.7 14.3 32 32 32s32-14.3 32-32V352l64.3-16.1c41.1-10.3 84.6-5.5 122.5 13.4c44.2 22.1 95.5 24.8 141.7 7.4l34.7-13c12.5-4.7 20.8-16.6 20.8-30V66.1c0-23-24.2-38-44.8-27.7l-9.6 4.8c-46.3 23.2-100.8 23.2-147.1 0c-35.1-17.6-75.4-22-113.5-12.5L64 48V32z" fill="currentColor" transform="translate(-224 -256)"></path></g></g></svg><span class="vue-fa-layers-text" style="font-weight:900;"><span style="height:1em;line-height:.75em;font-size:1.33333em;vertical-align:-.225em;transform-origin:center;overflow:visible;color:white;transform:translate(0em,0em) scale(-0.25,-0.25) rotate(-30deg);"><!--[--> NEW <!--]--></span></span><!--]--></span>